src/settings/StaffSlips/StaffSlipDetail.css

```diff
@@ -2,9 +2,3 @@
   border: 1px solid #ccc;
   padding-left: 5px;
 }
-
-.preview {
-  justify-content: flex-end;
-  padding-right: 5px;
-  margin-bottom: 5px;
-}
```

Comment:
I think we can remove this CSS file. I do not see the usage of styles from this file. 
Could you check it?

Reply:
looks like that is stale css which was not cleared earler.